pkg/image/mapping.go

```diff
@@ -14,10 +14,24 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// Format refers to the container image format.
+// It defines the structure of the image, and can be
+// * [Docker Image Manifest V2, Schema 1](https://docs.docker.com/registry/spec/manifest-v2-1/)
+// * [Docker Image Manifest V2, Schema 2](https://docs.docker.com/registry/spec/manifest-v2-2/)
+// * [OCI](https://github.com/opencontainers/image-spec)
+type Format string
+
+const (
+	OCIFormat      Format = "oci"
+	DockerV2Format Format = "docker-v2"
+	// OtherFormat is used when no analysis into the image is done to determine its format
+	OtherFormat Format = "other"
```

Comment:
Are these strings required? Would an enum work better so the "Other" valid does not have to be set?

Reply:
Great suggestion too @jpower432 
I just learned about Golang's enums